app/src/main/java/com/pvp/app/ui/screen/calendar/TaskFormsCreate.kt

```diff
@@ -143,6 +143,7 @@ fun TaskCreateForm(
     var duration by remember { mutableStateOf(Duration.ofMinutes(0)) }
     var reminderTime by remember { mutableStateOf<Duration?>(null) }
     var activity by remember { mutableStateOf(SportActivity.Walking) }
+    //var supportsDistanceMetrics by remember { mutableStateOf(false) }
```

Comment:
These are your changes and you aren't using this variable at all. Instead of commenting out and leaving them as is, you should completely remove them (4 lines in total) to not clutter code base with redundant comments (and/or code).